### PR TITLE
Add Django 3.0 to supported versions

### PR DESCRIPTION
**Pull request by lanshark**
Drop support for python 3.5

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+1.1.1 (2020-04-17)
+------------------
+* Add support for Wagtail >3.0.
```
Comment (MiltonLn) [inline CHANGELOG:6]: I think you made a typo here `Wagtail` maybe you meant `Django` right?

Comment (lanshark) [reply CHANGELOG]: Fixed!

**Comment by loleg** [comment]
Could you please fix the use of `{% load staticfiles %}` which [was removed](https://docs.djangoproject.com/en/dev/releases/3.0/#features-removed-in-3-0) in Django 3.0? E.g. warning being thrown:

```
Invalid template /usr/local/lib/python3.6/site-packages/puput/templates/wagtailadmin/pages/_editor_css.html: 'staticfiles' is not a registered tag library.
```

**Comment by lanshark** [comment]
> Could you please fix the use of `{% load staticfiles %}` which [was removed](https://docs.djangoproject.com/en/dev/releases/3.0/#features-removed-in-3-0) in Django 3.0? E.g. warning being thrown:
> 
> ```
> Invalid template /usr/local/lib/python3.6/site-packages/puput/templates/wagtailadmin/pages/_editor_css.html: 'staticfiles' is not a registered tag library.
> ```

Done!

**Comment by marctc** [comment]
@lanshark sorry i messed up deleting that. Do you mind to revert your changes on `CHANGELOG` and `__init__.py`?

**Comment by lanshark** [comment]
> @lanshark sorry i messed up deleting that. Do you mind to revert your changes on `CHANGELOG` and `__init__.py`?

I pulled your change, reverted it, and removed the changes to CHANGELOG.  It should be ready for you.

**Comment by marctc** [comment]
Can revert your changes on `__init__.py` as well?

**Pull request merged**